README.md

```diff
@@ -183,15 +183,20 @@ browserstack.idle.timeout=150
 browserstack.network.logs=true
 browserstack.project.name=local
 browserstack.build.name=local
+browserstack.gpsLocation=51.5074,0.1278
+app.token.url.v2=https://login.microsoftonline.com/<azure_app_id>/oauth2/v2.0/token
+app.client.id.v2=<azure_client_id>
+app.client.secret.v2=<azure_client_secret>
 app.username=<app_username>
 app.password=<app_userpass>
 app.token.url=https://login.microsoftonline.com/<azure_app_id>/oauth2/token
 app.client.id=<azure_client_id>
-base.path.url=https://api.develop.cvs.dvsacloud.uk/<branch_name>
+base.path.url=<test_environment_url>
 waitForQuiescence=true
 local.name=<your_name>
 automationName=XCUITest
 parallel.consumers=1
+app.users=<app_username><staff_id><password>
```

Comment:
The information related to why and how to use these fields will be taken care as part of KT. As I don't want a whole lot of explanation in the README file.

Reply:
Unfortunately, I disagree on this one as the code evolves every day (and will remain the single source of truth), most of the time confluence pages are not updated and the maintainers will be looking at the code repository and code most of the time in fact.
It is easier to search through the codebase rather than confluence.
It could be a simple comment next to the method where we process this data `getUsers()` ?

Reply:
changes done. Please review